### DISCUSSION TITLE
How do I assign an app to only one user? e.g. Multiple apps, but a user can only access the app assigned to them.

### DISCUSSION THREAD

**Discussion by sohailna** [Budibase/budibase#2910] [Help] [answered]
Hey everyone! Been playing around with Bb Cloud and it's slick! Great product 👏 

Just wanted to clarify one thing before I sink more time into it - I checked the Portal section of the docs but couldn't find an answer: 

The use case is to build a private dashboard for clients, each app is meant to be a private dashboard for each client and so each client should only be able to see their own dashboard aka app in the portal.

So, how do you give a user access to only 1 app?
I thought it might just be as simple as creating a user and assigning permissions for a specific app, or changing the permissions on an app by selecting what users can access it.

From what I could tell, there are groups of users and those groups have permissions. 

Can I create custom groups? Couldn't find anything in the docs in regards to User Groups.

If not, is my use case possible? i.e. multiple apps, but each user can only access the one assigned to them

Thanks!

**Comment by mjashanks** [comment, marked as answer]
Hi! Thanks for coming.

Right now, you can assign Roles per user, per app. Then, inside each app you can define what screens / data each role can see. However, everyone gets the "Basic" role by default, so can "see" all of the apps, even if they cant do anything inside the app.

By the end of this month, this will change. Users will get "No Access" by default. So, when a user is added, you would have to give them access to their app. Apps that they have "No Access" to will be completely invisible and inaccessible to them.

Sometime next year, we will be adding the concept of Groups, just as you describe. can't give you a date for it. When it comes, it will be a paid feature. We discuss all of this here: https://github.com/Budibase/budibase/discussions/2185.

**Reply by sohailna** [reply]
"No Access" should definitely do the job - thanks!

**Reply by pearcircuitmike** [reply]
Can I make a dashboard specific to each user, for example. I.E. a user signs up and then can CRUD their own entries (but they are isolated to just them and other users see only the DB entries they create)? @mjashanks

**Reply by mjashanks** [reply]
@pearcircuitmike - https://docs.budibase.com/guides/items-assigned-to-me - you can do the same using a "Created By" auto column.
